### PR TITLE
Fixed build Kepka on other than x86 architectures

### PR DESCRIPTION
**Pull request by xvitaly**
Fixed build Kepka on other than x86 architectures.

### CODE REVIEW

**Comment by crackedmind** [comment]
:white_check_mark: [Build kepka 1.0.75-cewnkwgi completed](https://ci.appveyor.com/project/procxx/tdesktop/build/1.0.75-cewnkwgi) (commit https://github.com/procxx/kepka/commit/91967f476b by @xvitaly)

File: Telegram/SourceFiles/config.h.in [hunk 1]
```diff
@@ -251,7 +251,7 @@ static const char *ApiHash = "344583e45741c457fe1862106095a5eb";
 #endif
 
 #if Q_BYTE_ORDER == Q_BIG_ENDIAN
-#error "Only little endian is supported!"
+#warning "Only little endian is supported!"
```
Comment (ForNeVeR) [inline Telegram/SourceFiles/config.h.in]: Wait, does it mean we support big endian now? Maybe delete the warning then?

Comment (xvitaly) [reply Telegram/SourceFiles/config.h.in]: We need to test it first.

Comment (ForNeVeR) [reply Telegram/SourceFiles/config.h.in]: Alright, then I suggest to change the text: "WARNING: Big endian support in Kepka is experimental"

It should be clear from the warning that
- we're working on it and we aren't sure if it works or not
- what application causes this warning (could be useful for cases when our code is transitive dependency of something else)

**Review by ForNeVeR — APPROVED** [review]

**Comment by crackedmind** [comment]
:white_check_mark: [Build kepka 1.0.75-wmqestfk completed](https://ci.appveyor.com/project/procxx/tdesktop/build/1.0.75-wmqestfk) (commit https://github.com/procxx/kepka/commit/c479aa6aaa by @xvitaly)

**Pull request merged**